accounts/urls.py

```diff
@@ -4,6 +4,7 @@
 
 
 urlpatterns = [
+    path('signup/', views.signup, name='signup'),
```

Comment:
This is correct, but there is now no entry point to your sign up page except manually typing in the url. So also change line 102 of `base.html` to redirect the signup reverse url from `"student_signup"`  to `"signup"`